### PR TITLE
feat: houston on zed

### PR DESCRIPTION
**Pull request by nermalcat69**
**Changes made**

- [x] Moved vscode's houston from `themes/houston.json` to `themes/vscode/houston.json`.
- [x] Made changes in package.json.
- [x] Added Zed extension in `themes/zed/~` (Includes: `extension.toml` & `houston.json`).

### CODE REVIEW

**Comment by nermalcat69** [comment]
Oh changeset wait

File: themes/zed/extension.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+id = "astro-build.houston"
+name = "Houston"
+version = "0.0.1"
+schema_version = 1
+authors = ["withastro"]
+description = "A Zed theme featuring cool blues, minty greens, and soft purples"
+repository = "https://github.com/withastro/"
```
Comment (at-the-vr) [inline themes/zed/extension.toml]:
```suggestion
repository = "https://github.com/withastro/houston-vscode"
```

**Review by at-the-vr — COMMENTED** [review]
Could not check the Theme cuz Windows User, leaving that to Community 😉

**Comment by namishh** [comment]
> Could not check the Theme cuz Windows User, leaving that to Community 😉

If you want a preview this is what it looks like 
image


The code editor and syntax highlighting is totally fine, though i believe the tab bar, the menu, scrollbar, and active panel buttons are still "unthemed". If needed I would be willing to contribute!😄 

Edit: With astro making official plugin for zed is neovim and helix also in the plannings?

**Comment by BMOit** [comment]
hey @chadcat7 i think @at-the-vr asked macOS users to test

**Comment by BMOit** [comment]
Also i tested it on my macOS machine, it works totally fine but not sure about windows :pray:

**Comment by at-the-vr** [comment]
Zed is yet to be released for Windows users 🤞really appreciate the approval for MacOS 🤝

**Comment by namishh** [comment]
> hey @chadcat7 i think @at-the-vr asked macOS users to test

I'm testing on Fedora Linux. Though I'm skeptical it would be a os related issue

**Comment by nermalcat69** [comment]
We need another macOS guy to test this out

**Comment by nermalcat69** [comment]
Too slow im closing this :/

**Pull request closed without merging**